service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json

```diff
@@ -62,25 +62,25 @@
       ],
       "defaultValue": "My",
             "metadata": {
-        "description": "The Store name where the cert will be deployed in the VM"
+     "description": "The Store name where the cert will be deployed in the VM" 
       }
     },
     "certificateThumbprint": {
       "type": "string",
             "metadata": {
-        "description": "Certificate Thumbprint"
+      "description": "Certificate Thumbprint"
       }
     },
     "sourceVaultValue": {
```

Comment:
Let's call this keyVaultResourceId to be more clear on the expectation here. sourceVaultValue doesn't really mean anything